.github/workflows/vulnerability-scanning.yml

```diff
@@ -5,15 +5,27 @@ on:
     branches: [ "master", "main" ]
   pull_request:
     branches: [ "master", "main" ]
+    types: [ opened, synchronize, ready_for_review ]
```

Comment:
Do we really need vuln scanning on *every* commit on *every* PR?
Is this sth that might be enough on "open" and "ready"? Or even on "ready" only?
```suggestion
    paths:
      - pom.xml
      - '**/pom.xml'
      - Dockerfile
```